qiskit/compiler/assemble.py

```diff
@@ -248,6 +251,12 @@ def _parse_pulse_args(backend, qubit_lo_freq, meas_lo_freq, qubit_lo_range,
                 meas_freq_est=backend_config_defaults.get('meas_freq_est')
             )
 
+        if meas_level not in backend_config.meas_levels:
```

Comment:
To cover backends which currently return `meas_levels=None`:

```suggestion
        if meas_level not in getattr(backend_config, 'meas_levels', [MeasLevel.CLASSIFIED]):
```